### DISCUSSION TITLE
duxapp：首个支持鸿蒙的 UI 库，同时还兼容 React Native、小程序、H5

### DISCUSSION THREAD

**Discussion by ShaoGongBra** [NervJS/taro#16983] [生态]
如何兼容的查看这篇文章
https://duxapp.cn/blog/harmony

开发文档
https://duxapp.cn/

**Comment by vaynevayne** [comment]
支持api 调用modal吗?

**Reply by ShaoGongBra** [reply]
那当然是支持的

**Reply by vaynevayne** [reply]
https://duxapp.com/docs/duxapp/component/ActionSheet/
定制不了ui, 扩展性不足

**Reply by ShaoGongBra** [reply]
这只是封装过的一个方法，你完全可以把自己的UI封装成api的形式调用
https://duxapp.com/docs/duxapp/component/TopView#addel-option
